kentikapi/client_integration_test.go

```diff
@@ -417,3 +425,110 @@ func timePtr(v time.Time) *time.Time {
 func durationPtr(v time.Duration) *time.Duration {
 	return &v
 }
+
+func TestGRPCClient_GetAgent(t *testing.T) {
+	tests := []struct {
+		name    string
+		id      string
+		res     *pb.GetAgentResponse
+		errCode codes.Code
```

Comment:
```suggestion
		expectedErrorCode codes.Code
```
Expected values should have _expected_ prefix for readability. Another option is e.g. _wantFoo_, but we use _expectedFoo_ in the project.

Reply:
Done